cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -252,10 +253,27 @@ instance HasSeverityAnnotation (TraceSendRecv a) where
 instance HasPrivacyAnnotation a => HasPrivacyAnnotation (TraceLabelPeer peer a)
 instance HasSeverityAnnotation a => HasSeverityAnnotation (TraceLabelPeer peer a)
 
+getFetchDecisionSeverityAnnotation :: FetchDecision a -> Severity
+getFetchDecisionSeverityAnnotation fd =
+    case fd of
+      Left FetchDeclineChainNotPlausible     -> Info
```

Comment:
```suggestion
      Left FetchDeclineChainNotPlausible     -> Debug
```
This happens whenever a peer has a shorter chain than us, i.e. all the time when we have more than one peer since they're not perfectly in sync.